tests/baselines/reference/fatarrowfunctionsOptionalArgsErrors2.errors.txt

```diff
@@ -18,16 +21,22 @@ tests/cases/compiler/fatarrowfunctionsOptionalArgsErrors2.ts(4,17): error TS1005
 tests/cases/compiler/fatarrowfunctionsOptionalArgsErrors2.ts(4,20): error TS2304: Cannot find name 'a'.
 
 
-==== tests/cases/compiler/fatarrowfunctionsOptionalArgsErrors2.ts (18 errors) ====
+==== tests/cases/compiler/fatarrowfunctionsOptionalArgsErrors2.ts (21 errors) ====
     var tt1 = (a, (b, c)) => a+b+c;
-                  ~
-!!! error TS1003: Identifier expected.

```

Comment:
hmm. so we had this test originally? why did not we catch this the first time around then?

Reply:
I noticed the baseline change but (1) I thought that removal of 3 errors was actually an improvement (2) didn't think it was indicative on any further problem. I also looked at the emitted code and thought it was better -- closer to what was written at least.

Reply:
well.. the new error is a parse error. so a bit worse than the 3 semantic ones.